index.js

```diff
@@ -7,7 +7,7 @@ var path       = require('path');
 var fs         = require('fs');
 var stringify  = require('json-stable-stringify');
 var mergeTrees = require('broccoli-merge-trees');
-var funnel = require('broccoli-funnel');
+var funnel     = require('broccoli-funnel');
```

Comment:
See https://github.com/babel/broccoli-babel-transpiler/blob/master/index.js#L51. I just aligned it.